ext/pcntl/tests/pcntl_rfork_nowait.phpt

```diff
@@ -15,7 +15,7 @@ if ($pid > 0) {
 	var_dump($pid);
 } else {
 	var_dump($pid);
-  sleep(1);
+  sleep(2);
```

Comment:
Wouldn't it make more sense to sleep before calling var_dump($pid)? If not, could you document the reason for sleep going after the var_dump is called?

It seems like if sleep was called before var_dump, it'd be more likely you'd get int(%d) and int(0) in that order.

```php
} else {
   sleep(1);
   var_dump($pid);
}
```

Reply:
is intentional since there is no pcntl_wait use here otherwise the child does not appear.